Assignment4/JorgeOP46n2-180113/task06.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+"""Task06.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1hSv61ejXESrp34CUz1PqZCsoA43tzc8y
+
+**Task 06: Modifying RDF(s)**
+"""
+
+!pip install rdflib 
```

Comment:
This will not run in Python, you need to comment out this line. The same applies for other files